content/cloud-computing/concepts/iaas/iaas.md

```diff
@@ -0,0 +1,18 @@
+---
+Title: 'IaaS'
+Description: 'Infrastructure as a service (IaaS) is a type of cloud computing service offering remote computer, storage and networking resources that are available on demand'
+Subjects:
+  - 'Computer Science'
+  - 'Networking'
+Tags:
+  - 'Networking'
+  - 'Cloud'
+  - 'Infrastructure'
+CatalogContent:
+  - 'foundations-of-cloud-computing'
+  - 'paths/back-end-engineer-career-path'
+---
+
+**Infrastructure as a service (IaaS)** is a type of cloud computing service offering remote computer, storage and networking resources that are available on demand. These services are typically offered on a pay-as-you-go basis. The user is given control of virtual machines that are guests on a cloud server running a hypervisor such as Oracle VirtualBox or VMware ESXi. They are allowed to run arbitrary software on these virtual machines, including operating systems. They are also provided with an API to control the low-level details of the underlying network infrastructure: things such as backups, partitioning, security, physical resources and so on. While the user does not manage or control the underlying cloud infrastructure, they have granular control over operating systems, storage, deployed applications, as well as some networking components such as host firewalls.
```

Comment:
```suggestion
**Infrastructure as a service (IaaS)** is a cloud computing model that offers remote computer, storage, and networking resources that are available on demand. These services are typically offered on a pay-as-you-go basis. The user is given control of virtual machines that are guests on a cloud server running a hypervisor such as Oracle VirtualBox or VMware ESXi. They are allowed to run arbitrary software on these virtual machines, including operating systems.

IaaS is also provided with an [API](https://www.codecademy.com/resources/docs/general/api) to control the low-level details of the underlying network infrastructure: things such as backups, partitioning, security, physical resources, and so on. While the user does not manage or control the underlying cloud infrastructure, they have granular control over operating systems, storage, deployed applications, as well as some networking components such as host firewalls.
```

Reply:
Given that we're working on a separate "Types of Cloud" entry in another PR, I think we should avoid this phrasing for these entries to mitigate any possible confusion.